exporter/exporter_test.go

```diff
@@ -34,9 +34,37 @@ var _ = Describe("CheckForNewApps", func() {
 		Eventually(fakeWatcherManager.AddWatcherCallCount).Should(Equal(1))
 	})
 
+	It("does not create a new appWatcher if the app state is stopped", func() {
+		fakeClient.ListAppsByQueryReturns([]cfclient.App{
+			{Guid: "33333333-3333-3333-3333-333333333333", Instances: 1, Name: "foo", SpaceURL: "/v2/spaces/123", State: "STOPPED"},
+		}, nil)
+
+		e := exporter.New(fakeClient, fakeWatcherManager)
+
+		go e.Start(100 * time.Millisecond)
+
+		Eventually(fakeWatcherManager.AddWatcherCallCount).Should(Equal(0))
+	})
+
+	It("creates a new appWatcher if a stopped app is started", func() {
+		fakeClient.ListAppsByQueryReturnsOnCall(0, []cfclient.App{
+			{Guid: "33333333-3333-3333-3333-333333333333", Instances: 0, Name: "foo", SpaceURL: "/v2/spaces/123", State: "STOPPED"},
```

Comment:
is this pairing of `Instances:0, State:"STOPPED"` and `Instances:1, State:"STARTED"` representative of what we see from the API?

Reply:
Yes 💯 Have tested all locally and are confident.